src/test/resources/net/sf/jabref/logic/exporter/MsBibExportFormatTest7.bib

```diff
@@ -0,0 +1,15 @@
+% Encoding: UTF-8
+
+@InBook{Orlowski;2011,
```

Comment:
Please name the test files after what they do and minimize them, not like the Stupro did just enumerating them :smile: